color.html

```diff
@@ -69,6 +73,10 @@
 
       /* Grid tweaks */
       ---material-grid-row-hover-background-color: rgba(255, 255, 255, 0.08);
+
+      /* Split layouts */
+      --material-split-layouts-splitter-color: rgba(255, 255, 255, 0.8);
+      --material-split-layouts-handle-color: rgba(0, 0, 0, 1.0);
```

Comment:
About naming the custom properties: use the element name, but replace `vaadin` with `material` (like you have). Here you have “split-layout**s**”. 

Also, these are internal tweak properties, not part of the public styling API, so they should be prefixed with 3 dashes, not just 2. So it should be `---material-split-layout-*`. At some point, we should rename them to `--_material-*` to make it more explicit that they are internal (this is already done in Valo).

Then about this change specifically. We should only add one tweak, `---material-split-layout-splitter-background-color`. The other color (the handle) can use `--material-body-background-color`.

Please update the comment above also to “Split layout tweaks”, so it’s consistent with the other comments.